prow/plugins/bugzilla/bugzilla_test.go

```diff
@@ -802,6 +814,10 @@ Instructions for interacting with me using PR comments are available [here](http
 			expectedLabels: []string{"bugzilla/valid-bug"},
 			expectedComment: `org/repo#1:@user: This pull request references [Bugzilla bug 123](www.bugzilla/show_bug.cgi?id=123), which is valid.
 
+Validations run on this bug:
+	- dependent bug [Bugzilla bug 124](www.bugzilla/show_bug.cgi?id=124) is in the state VERIFIED, which is one of the valid states (VERIFIED)
+	- bug has dependents
```

Comment:
This is only an implicit requirement, right?

Reply:
Yes, but it's a validation we run, so I added it to the list. We can remove it if you think it's unnecessary.